### ISSUE TITLE
Crash on startup

### ISSUE THREAD

**Issue by Rozbiynk** [NimajnebEC/auto-elytra#16]
https://mclo.gs/ak9fsUJ

**Comment by asundrus** [comment]
> https://mclo.gs/ak9fsUJ

Did you even look at the content of your link?

**Comment by Rozbiynk** [comment]
> > https://mclo.gs/ak9fsUJ
> 
> Did you even look at the content of your link?

yes, I look at that, it contain game crash on startup. What's wrong?
